src/moscot/base/problems/birth_death.py

```diff
@@ -159,46 +148,77 @@ def apoptosis_key(self: BirthDeathProtocol, key: Optional[str]) -> None:
         self._apoptosis_key = key
 
 
-@d.dedent
 class BirthDeathProblem(BirthDeathMixin, OTProblem):
-    """Optimal transport problem which allows to estimate the marginals with a birth-death process.
+    """:term:`OT` problem used to estimate the :term:`marginals` with the
+    `birth-death process <https://en.wikipedia.org/wiki/Birth%E2%80%93death_process>`_.
 
     Parameters
     ----------
-    %(adata_x)s
-    """
+    args
+        Positional arguments for :class:`~moscot.base.problems.OTProblem`.
+    kwargs
+        Keyword arguments for :class:`~moscot.base.problems.OTProblem`.
+    """  # noqa: D205
 
     def estimate_marginals(
-        self: BirthDeathProblemProtocol,
+        self,  # type: BirthDeathProblemProtocol
         adata: AnnData,
         source: bool,
         proliferation_key: Optional[str] = None,
         apoptosis_key: Optional[str] = None,
-        marginal_kwargs: Mapping[str, Any] = MappingProxyType({}),
-        **_: Any,
+        **kwargs: Any,
     ) -> ArrayLike:
-        """TODO."""
+        """Estimate the source or target :term:`marginals` with the
+        `birth-death process <https://en.wikipedia.org/wiki/Birth%E2%80%93death_process>`_,
+        as suggested in :cite:`schiebinger:19`.
```

Comment:
it does not only implement this method, but also a simpler one. I will adapt the docstrings.